cypress/page-objects/lib/ui-element.ts

```diff
@@ -32,4 +42,31 @@ export class UIElement {
   haveText(value: string) {
     this.getElement().should('have.text', value)
   }
+
+  getFilter(text: string) {
```

Comment:
I'd say it's not a method for UIElement. It fits better for some custom UIWidget.

Reply:
moved to Button